examples/decoding/plot_linear_model_patterns.py

```diff
@@ -20,15 +20,18 @@
 """
 # Authors: Alexandre Gramfort <alexandre.gramfort@telecom-paristech.fr>
 #          Romain Trachel <trachelr@gmail.com>
+#          Jean-Remi King <jeanremi.king@gmail.com>
```

Comment:
Ok, I also updated the references of all decoding examples, following the rsa example. LMK whether it's ok.